src/components/Navigation.jsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { HumanLoggedIn } from './Icons';
 
 function Navigation() {
   return (
-    <header>
-      <h1>Bookstore</h1>
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/categories">Categories</Link>
-          </li>
-        </ul>
-      </nav>
-    </header>
+    <nav>
+      <ul>
+        <li>
+          <h1 className="bookstore-cms">Bookstore CMS</h1>
+        </li>
+        <li>
+          <Link to="/" className="books-nav">BOOKS</Link>
+        </li>
+        <li>
+          <Link to="/categories" className="categories-nav">CATEGORIES</Link>
```

Comment:
- [ ] Well done! Your `categories-nav` exhibits the proper font size and color. However, according to the design, it should also have an opacity value of `0.5`. Kindly consider adding this opacity to the element to achieve the desired look.

- [ ] Moreover, your `books-nav` and `categories-nav` share similar styles, differing only in opacity. To avoid redundant properties, I kindly suggest consolidating these common styles within the `books-nav` class. then add the `books-nav` alongside the `categories-nav` in this link. This approach will streamline your code and eliminate unnecessary repetition of properties.

![image](https://github.com/AliRezaBmeDu/bookstore/assets/55840999/dd70bb5e-8e82-452a-89ae-5948fd4e08d0)

First, delete the same attributes from the `categories-nav` class. then:
```suggestion
          <Link to="/categories" className="books-nav categories-nav">CATEGORIES</Link>
```
Your attention to these details will contribute to cleaner and more efficient styling in your project. Keep up the great work! 🎉🎨